wurst/util/StringUtils.wurst

```diff
@@ -157,6 +126,101 @@ public function string.reduce(string reduced) returns string
 public function string.reduce() returns string
 	return this.reduce(" ")
 
+public interface ForStringSplitCallback
+	function callback(string str)
+
+/** Custom Function which can split strings and runs callback for each string.
+	It can run even with no delimiter present (with single element).
+	Example usage:
+	"H001,H002".forEachIn(",") ->
+		doSomething()  */
+public function string.forEachIn(string delim, ForStringSplitCallback cb)
+	if this.length() > 0
+		int i = 0
+		int pos = this.indexOf(delim, i)
+
+		if pos == -1
+			cb.callback(this.substring(i, this.length()))
+
+		while pos != -1
+			cb.callback(this.substring(i, pos))
+			pos += 1
+			i = pos
+			pos = this.indexOf(delim, pos)
+			if pos == -1
+				cb.callback(this.substring(i, this.length()))
+
+/** Creates list of IDs converted from Int to their String value delimited by comma (using toRawCode).*/
+public function createDelimitedString(vararg int ids) returns string
+	string s = ""
+	var paramCount = 0
+	for id in ids
+		paramCount++
+		s += id.toRawCode()
+		s += ","
+	if s != ""
+		s = s.substring(0, s.length()-1)
+	if paramCount > WC3LIMIT_FUNCTIONS_PARAMETER_COUNT
+		error(WC3LIMIT_FUNCTIONS_PARAMETER_COUNT_ERROR)
+	return s
+
+/** Creates one long String from all Strings delimited by comma.*/
+public function createDelimitedString(vararg string ids) returns string
```

Comment:
why is this an extra function? isn't it the same as `joinBy(",")` ?

Reply:
joinBy is used to convert linkedlist to delimited string. This is creating delimited string from bunch of strings.

Reply:
Yea sry, I think I ment `commaList`.